autolift-scalaz/src/main/scala/autolift/scalaz/package.scala

```diff
@@ -0,0 +1,12 @@
+package autolift
+
+import _root_.scalaz.Functor
+import _root_.scalaz.Scalaz._
+
+package object scalaz extends autolift.AutoLifts {
```

Comment:
Maybe I will run into the same (or other) issues. I will update this PR if I have something better